app/helpers/application_helper/button/basic.rb

```diff
@@ -12,6 +12,12 @@ def initialize(view_context, view_binding, instance_data, props)
     end
   end
 
+  def role_allows_feature?
+    return role_allows?(:feature => self[:child_id]) unless self[:child_id].nil?
+    return role_allows?(:feature => self[:id]) if self[:items].nil? || self[:items].empty?
+    true
```

Comment:
Can you explain in plain English exactly what the intent is here? I'm having trouble understanding the intent, and very concerned because I'm wary of exceptions to how permissions work *outside of RBAC*. We just refactored a bunch of things in to RBAC because they were spread out all over the codebase. No exceptional behavior should exist in the UI, if I'm reading this correctly (check or don't check RBAC seems valid for basic vs. permissioned buttons, though)